packages/design-system-tokens/sketch/cmsds-token-importer/src/text.js

```diff
@@ -47,6 +61,10 @@ function parseFontWeight(tokenValue) {
 }
 
 function parseFontFamily(tokenValue) {
+  if (tokenValue === 'inherit') {
+    return null;
+  }
+
   // Only return the first one, and remove all quotes
   return tokenValue.split(',')[0].replaceAll('"', '').replaceAll("'", '').trim();
```

Comment:
does this need 2 `replaceAll`?

Reply:
One finds double quotes around font families, and one finds single quotes. I could use one regex, I guess

Reply:
oh i didn't see the double vs single there 😅 disregard!